### PR TITLE
Fix ESM bug, remove private dep

### LINKED ISSUE #8: Error [ERR_REQUIRE_ESM]: lodash-es
Hi,

i'm trying to use this plugin in my project and I've followed the instructions of the plugin.
I'm using `vite` version 4.0.1 and I've installed latest version of `vite-plugin-i18next-loader` which is 2.0.2.

This is my `vite.config.ts` file:
```ts

import react from '@vitejs/plugin-react';
import { resolve } from 'path';
import { defineConfig } from 'vite';
import i18nextLoader from 'vite-plugin-i18next-loader';
import mkcert from 'vite-plugin-mkcert';
import svgrPlugin from 'vite-plugin-svgr';
import viteTsconfigPaths from 'vite-tsconfig-paths';

import * as baseTheme from './src/styles/baseTheme.json';

export default defineConfig({
  server: {
    https: true,
    port: 3000,
    open: true,
  },
  build: {
    outDir: 'build',
  },
  css: {
    preprocessorOptions: {
      less: {
        modifyVars: {
          // Color Palettes
          '@blue-1': baseTheme.colorPalettes.daybreakBlue['blue-1'],
          ...
        },
        javascriptEnabled: true,
      },
    },
  },
  plugins: [
    react(),
    viteTsconfigPaths(),
    mkcert(),
    svgrPlugin(),
    i18nextLoader({
      paths: ['./src/locales/en'],
    }),
  ],
});
```

When I try to start my project with `vite` I get this error:

failed to load config from `./vite.config.ts`
error when starting dev server:
Error [ERR_REQUIRE_ESM]: require() of ES Module /node_modules/lodash-es/lodash.js from /node_modules/vite-plugin-i18next-loader/dist/index.cjs not supported.
Instead change the require of lodash.js in /node_modules/vite-plugin-i18next-loader/dist/index.cjs to a dynamic import() which is available in all CommonJS modules.
    at Object._require.extensions.<computed> [as .js] (file:///node_modules/vite/dist/node/chunks/dep-5605cfa4.js:63157:17)
    at Object.<anonymous> (/node_modules/vite-plugin-i18next-loader/dist/index.cjs:1:666)
    at Object._require.extensions.<computed> [as .js] (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:63157:17)
    at Object.<anonymous> (/vite.config.ts:35:49)
    at Object._require.extensions.<computed> [as .js] (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:63154:24)
    at loadConfigFromBundledFile (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:63162:21)
    at loadConfigFromFile (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:63020:34)
    at async resolveConfig (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:62643:28)
    at async createServer (file://node_modules/vite/dist/node/chunks/dep-5605cfa4.js:61943:20)
    at async CAC.<anonymous> (file://node_modules/vite/dist/node/cli.js:707:24)
    
  
Thank you

### PR DESCRIPTION
**Pull request by agustif**
Fixes #8

### CODE REVIEW

**Comment by jamespearson** [comment]
Please approve and release this.

**Comment by gamegee** [comment]
@rosskevin good for you to merge ?

**Comment by agustif** [comment]
@gamegee @jamespearson until it gets merged feel free to use this one https://www.npmjs.com/package/vite-plugin-i18next-loader-cjs

**Comment by rosskevin** [comment]
I'm sorry, I didn't get a notification on this!  Only the comment. Reviewing now...

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "vite-plugin-i18next-loader",
-  "version": "2.0.2",
+  "name": "vite-plugin-i18next-loader-cjs",
+  "version": "2.0.4",
```
Comment (rosskevin) [inline package.json:3]: Don't bump the version, `auto` will do that instead.  Also change back the name.

**Review by rosskevin — CHANGES_REQUESTED** [review]

**Comment by rosskevin** [comment]
I'm surprised that `auto` didn't provide a usable release from this PR...I'll have to look at the settings.

File: package.json (hunk 1 above)
Comment (agustif) [reply package.json:3]: Hey sure that makes sense!

**Comment by agustif** [comment]
> I'm surprised that `auto` didn't provide a usable release from this PR...I'll have to look at the settings.

I didn't knew about auto when I opened the PR, I was mainly trying to fix it to use myself. You suggested to change back from lodash-es to lodash. That's what was fixing the issue! Any ideas how to solve properly?

File: package.json [hunk 2]
```diff
@@ -56,12 +56,11 @@
...
     "js-yaml": "^4.1.0",
-    "lodash-es": "^4.17.21",
+    "lodash": "^4.17.21",
     "marked": "^4.1.1",
     "marked-terminal": "^5.1.1"
   },
   "devDependencies": {
-    "@alienfast/eslint-config": "^4.0.5",
     "@types/js-yaml": "^4.0.5",
     "@types/lodash-es": "^4.17.6",
```
Comment (gamegee) [inline package.json:65]: Also don't forget to replace by `@types/lodash`

**Comment by rosskevin** [comment]
@agustif I looked into why a canary release wasn't added to this PR by `auto`, and it's because you included `skip ci` in your last message (HEAD) before you PR'd.  If your latest commit was pushed - then you would have a canary to try.

I'm going to take a look at opening our eslint config so we can keep it.  I'd prefer it.  Give me a couple of hours to take a look at that - I should get this resolved today.

**Comment by rosskevin** [comment]
Github package is public now for `@alienfast/eslint-config`, I'll take a shot at this myself in another PR and we can try it out.

**Comment by rosskevin** [comment]
Closing in favor of my PR to solve this #10.  I need the canary to be tested before I'll release it.  Please take a look and test it out and comment there.

**Pull request closed without merging**